components/description-list/demo/description-list.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<meta charset="UTF-8">
+		<link rel="stylesheet" href="../../demo/styles.css" type="text/css">
+		<script type="module">
+			import '../../demo/demo-page.js';
+
+			import './description-list-test.js';
+		</script>
+	</head>
+	<body unresolved>
+		<d2l-demo-page page-title="d2l-description-list">
+
+			<h2>Vanilla semantics</h2>
+			<d2l-demo-snippet fullscreen>
+				<dl>
+					<dt>Course Name</dt>
+					<dd>Brightspace 101B A</dd>
+				</dl>
+			</d2l-demo-snippet>
+
+			<h2>Short</h2>
+			<d2l-demo-snippet fullscreen>
+				<d2l-test-dl></d2l-test-dl>
```

Comment:
Really unfortunate that we can't show actual usage in these demos - as I understand it, the blocker is that we can't apply styles to nested slotted children, so we need to import the styles and use them in a (Lit) component.

Reply:
Yeah, this is unfortunately one of the biggest drawbacks to using a wrapper component like we do here. This is how `d2l-table-wrapper` handles its demos as well:

https://github.com/BrightspaceUI/core/blob/8dc2e75b16f6ab9b95680667faaaae6cbfb557a8/components/table/demo/table.html#L19

Reply:
Yeah it's unfortunate. These demos are perhaps viewed less by others now that we have the Daylight site, but if we were really inclined to show something informative in the core demo page, we could:

```html
<d2l-demo-snippet code-view-hidden>
    <!-- the stuff -->
</d2l-demo-snippet>
<d2l-code-view language="javascript">
    // the stuff we really want to show
</d2l-code-view>
```

We could even add slot for it if we wanted it to look exactly the same in terms of layout.  But I feel like it's not really worth it because it becomes a maintenance issue and these demos are more for our own debugging/development purposes now.